CGF.py

```diff
@@ -0,0 +1,13 @@
+
+#given a list find the greatest commmon factor,( the largest number that is divisible by all 
```

Comment:
I think this problem statement doesn't make sense to me, where did you get it from? Greatest common factor is the _largest number that divides all items in the list_. In your problem statement it asks for "largest number that is divisible by all items in the list", which does not exist (i.e. it's the same as asking what's the largest number possible).